src/dehtml.rs

```diff
@@ -22,18 +22,18 @@ enum AddText {
     YesPreserveLineEnds,
 }
 
-// dehtml() returns way too many newlines; however, an optimisation on this issue is not needed as
-// the newlines are typically removed in further processing by the caller
-pub fn dehtml(buf: &str) -> String {
+/// dehtml() returns way too many newlines; however, an optimisation on this issue is not needed as
```

Comment:
Doc comments are show in HTML documentation.

This is not a proper documentation, just a note on implementation.